docs/openapi.yaml

```diff
@@ -156,6 +156,46 @@ paths:
                 items:
                   $ref: '#/components/schemas/Publication'
 
+  /{publicationIdentifier}/doi:
+    post:
+      summary: Reserves doi for draft publication
+      parameters:
+        - in: path
+          name: publicationIdentifier
+          schema:
+            type: string
+            format: uuid
+          required: true
+          description: UUID identifier of the Publication to fetch.
+      security:
+        - CognitoUserPool: [ 'https://api.nva.unit.no/scopes/backend', 'https://api.nva.unit.no/scopes/frontend','aws.cognito.signin.user.admin' ]
+      responses:
```

Comment:
What of:
- Illegal state. Publication not in draft mode?
- Illegal state. Doi already reserved?

Reply:
Skal ikke man returnere 200 OK hvis DOI allerede eksisterer?

Reply:
går for 200 etter en prat med Rurik og Anette